retriever/lib/datapackage.py

```diff
@@ -160,7 +160,13 @@ def create_json():
 
     contents['title'] = clean_input("title: ", ignore_empty=True)
     contents['description'] = clean_input("description: ", ignore_empty=True)
-    contents['citation'] = clean_input("citation: ", ignore_empty=True)
+    citations = clean_input("citations (separated by ';'): ",
+                            split_char=';', ignore_empty=True)
+    contents['citation'] = ""
+    if len(citations) == 1:
```

Comment:
I don't get what you are trying to check here
```python
>>> c="a"
>>> len(c)
1
```

Reply:
@henrykironde, `citations` would return a list of user entered citations. If the user entered only a single citation, then instead of storing a list of only one element, the `citations[0]` would be stored. Else, the entire list would be stored.

Reply:
In this case, I think it is good to assume that or enforce all new scripts' citations as lists.

Reply:
In which case we may not need any checking

Reply:
@henrykironde done.